src/Bicep.Core/Syntax/ImportDeclarationSyntax.cs

```diff
@@ -1,45 +1,57 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System.Collections.Generic;
-using System.Linq;
+using Bicep.Core.Extensions;
 using Bicep.Core.Navigation;
 using Bicep.Core.Parsing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Bicep.Core.Syntax
 {
-    public class ImportDeclarationSyntax : StatementSyntax, ITopLevelNamedDeclarationSyntax
+    public class ImportDeclarationSyntax : StatementSyntax, ITopLevelDeclarationSyntax
     {
-        public ImportDeclarationSyntax(IEnumerable<SyntaxBase> leadingNodes, Token keyword, IdentifierSyntax providerName, SyntaxBase asKeyword, IdentifierSyntax aliasName, SyntaxBase config)
+        private readonly Lazy<ImportSpecification> lazySpecification;
+
+        public ImportDeclarationSyntax(IEnumerable<SyntaxBase> leadingNodes, Token keyword, SyntaxBase specificationString, ImportWithClauseSyntax? withClause, ImportAsClauseSyntax? asClause)
             : base(leadingNodes)
         {
-            AssertKeyword(keyword, nameof(keyword), LanguageConstants.ImportKeyword);
```

Comment:
>AssertKeyword [](http://example.com/codeflow?start=12&length=13)

Why remove the asserts? We have them so you can't construct invalid syntax nodes manually.

Reply:
Add them back.